src/Libraries/Microsoft.Extensions.Diagnostics.Testing/Logging/FakeLogger.cs

```diff
@@ -66,8 +66,8 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
 
         var l = new List<object?>();
         ScopeProvider.ForEachScope((scopeState, list) => list.Add(scopeState), l);
-
-        var record = new FakeLogRecord(logLevel, eventId, ConsumeTState(state), exception, formatter(state, exception),
+        var (stateTypes, consumedState) = ConsumeTState(state);
```

Comment:
As per our coding style guide, we don't use `var` unless the type is obvious.
https://github.com/dotnet/extensions/blob/main/CONTRIBUTING.md#dos-and-donts